### ISSUE TITLE
Question about the leave.py and total.py

### ISSUE THREAD

**Issue by Fcz-codes** [yihuacheng/IVGaze#7]
Thank you very much for sharing, I am very interested in your work. Your research on gaze estimation often uses the two files leave.py and total.py, but the data in the train and test yaml files are the same, and there is no distinction between the training set and the test set. Why is this?
